### PR TITLE
MP 6.0 draft results

### PR DESCRIPTION
**Pull request by tevans78**

### CODE REVIEW

File: microprofile/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            telemetry/1.0/
+                22.0.0.13-beta-Telemetry-1.0-Java11-TCKResults.adoc
+                22.0.0.13-beta-Telemetry-1.0-Java17-TCKResults.adoc
+    ```
+
+3. Draft versions of these results files can be found in the "TCK Results" downloads for each build. Note that these are only draft/templates and will need to be updated manually.
+    - If the results are being taken from a build which has been SOE'd then there will be results for a large number of platforms. You should choose a consistent set of results from just one platform, preferably running with a Semeru JVM. e.g. `RHEL8_X86_IBM_SEMERUJDK11_CERTIFIED_64_EBC`
+    - Make sure that the Liberty version is correct if using the beta rather than GA version
+    - Ensure that all links within the results are correct.
+    - In some cases you may have to make an educated guess about what the URL will be when released.
```
Comment (Azquelt) [inline microprofile/README.md:38]: Is this a complete list of things that need to be updated? At the moment it's a bit vague.

Comment (Azquelt) [reply microprofile/README.md:38]: Also, with this file it its current location, will it be published to the website?

**Review by Azquelt — COMMENTED** [review]
The new MP certifications look good. I'm not sure about putting the README in the microprofile folder, I'd like to be sure it won't get published to the site (either as the raw md or as rendered HTML).

I'm not happy that the commit from #207 is included and has changes in it, rather than just being a merge commit. This suggests that prod and staging are out of sync. That's a separate issue so I'd suggest rebasing this PR on the previous commit (1cedb028) and getting the desync sorted out separately.

**Review by mbroz2 — APPROVED** [review]

File: microprofile/README.md (hunk 1 above)
Comment (mbroz2) [reply microprofile/README.md:38]: @Azquelt this was a good concern and to answer the question, ol.io explicitly doesn't build any `*.md` files.  You can see the list of exclusions here:  https://github.com/OpenLiberty/openliberty.io/blob/e5be04357c559ba0247a8b835ce7e031ce6e5035/src/main/content/_config.yml#L47

**Pull request merged**